microsoft-edge/toc.yml

```diff
@@ -218,7 +218,11 @@
 
                 - name: Map preprocessed code to source code
                   href: devtools-guide-chromium/javascript/source-maps.md
-                  displayName: Sources tool
+                  displayName: Sources tool, source maps
+
+                - name: Debug original code using the Azure Artifacts symbol server
+                  href: devtools-guide-chromium/javascript/ado-symbol-server.md
```

Comment:
Rendered page URL:
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/javascript/ado-symbol-server?branch=pr-en-us-1766

dk why I'm seeing this as an orphaned page.  Will try Refresh.

Reply:
The orphan TOC was b/c loaded PR 1768's page, had to refresh to reload 1766 TOC.